### ISSUE TITLE
Update images to fix CVEs

### ISSUE THREAD

**Issue by jimangel** [Dynatrace/dynatrace-oneagent-operator#102]
Before I can promote a docker image to production, it need to pass a security scan with no medium or high vulns. In the case of **dynatrace/oneagent:latest** - it trips with:

name | version | CVE | level | solution
-- | -- | -- | -- | --
openssl | 1.1.1a-r1 | CVE-2019-0190 | medium | 
openssl | 1.1.1a-r1 | CVE-2019-1543 | medium | Upgrade package openssl to version 1.1.1b-r1 or above.

It appears this image is built with Alpine 3.9

```
sh-4.4# cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.9.2
PRETTY_NAME="Alpine Linux v3.9"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"
sh-4.4# openssl version
OpenSSL 1.1.1a  20 Nov 2018
```

I _think_ this is fixed simply with a rebuild using the latest alpine 3.9.3 base image since that openssl version should now be [1.1.1b-r1](https://pkgs.alpinelinux.org/packages?name=openssl&branch=v3.9)

**dynatrace/dynatrace-oneagent-operator:v0.3.0**:

name | version | CVE | level | solution
-- | -- | -- | -- | --
busybox | 1.28.4-r3 | CVE-2018-20679 | medium |

This should be a quick fix by upgrading Alpine to 3.9 (.3) in your [Dockerfile ](https://github.com/Dynatrace/dynatrace-oneagent-operator/blob/master/build/Dockerfile) as that should contain a [patched - 1.29.3-r10](http://bld2.alpinelinux.org/distfiles/buildlogs/build-3-9-x86/main/busybox/busybox-1.29.3-r10.log) version of busybox.

**Comment by esukram** [comment]
Thank you for reporting that issue. We will take care!

**Comment by esukram** [comment]
@jimangel I will forward your finding internally, as this is a different container image. You are referring to the actual OneAgent container image. This repo is _just_ for the operator.

But definitely something we have to tackle.

**Comment by esukram** [comment]
Fixed with #103 and #104 respectively.
